### DISCUSSION TITLE
I can not add a collaborator to the private repo

### DISCUSSION THREAD

**Discussion by Yakimochkina** [community/community#23837] [New to GitHub]
<p>Hi!<br>
</p><div class="lightbox-wrapper"><a class="lightbox" href="https://aws1.discourse-cdn.com/github/original/3X/a/3/a3232d6cda283f811e64b9d13f680456530d536c.png" data-download-href="/uploads/short-url/nhbc5Bh8RRiX3L4lvqEyaENwPta.png?dl=1" title="Screen Shot 2021-10-04 at 16.50.39" rel="noopener nofollow ugc"><img src="https://user-images.githubusercontent.com/39552859/181079645-d5dcb1ec-dca8-4240-807e-f8448e49eee0.png" alt="Screen Shot 2021-10-04 at 16.50.39" data-base62-sha1="nhbc5Bh8RRiX3L4lvqEyaENwPta" width="690" height="396" srcset="https://aws1.discourse-cdn.com/github/optimized/3X/a/3/a3232d6cda283f811e64b9d13f680456530d536c_2_690x396.png, https://aws1.discourse-cdn.com/github/optimized/3X/a/3/a3232d6cda283f811e64b9d13f680456530d536c_2_1035x594.png 1.5x, https://aws1.discourse-cdn.com/github/optimized/3X/a/3/a3232d6cda283f811e64b9d13f680456530d536c_2_1380x792.png 2x" data-small-upload="https://aws1.discourse-cdn.com/github/optimized/3X/a/3/a3232d6cda283f811e64b9d13f680456530d536c_2_10x10.png"><div class="meta">
<svg class="fa d-icon d-icon-far-image svg-icon" aria-hidden="true"><use xlink:href="#far-image"></use></svg><span class="filename">Screen Shot 2021-10-04 at 16.50.39</span><span class="informations">1814×1042 118 KB</span><svg class="fa d-icon d-icon-discourse-expand svg-icon" aria-hidden="true"><use xlink:href="#discourse-expand"></use></svg>
</div></a></div>
<p>I can not add a collaborator to the private repo. No chance to find a person.</p>
<p>I can not contact support promptly. What to do?</p>

**Comment by tajmone** [comment]
<p>That’s strange, you should be able to add collaborators to private repositories even with a Free account (although not many).</p>
<p>It looks like the problem is in the WebUI not showing users. I’ve recently added people to my own repositories, and from what I remember the list of people should start to fill in as you type, unless you’ve types a username that doesn’t exist (in which case the list is empty and the confirmation button get greyed out).</p>
<p>Same problem reported here:</p>
<aside class="quote" data-post="2" data-topic="204684">
  <div class="title">
    <div class="quote-controls"></div>
    
    <a href="https://github.community/t/cannot-add-collabolator-to-repository/204684/2">Cannot add collabolator to repository</a> <a class="badge-wrapper  bullet" href="/c/github-help/48"><span class="badge-category-bg" style="background-color: #6a737d;"></span><span style="" data-drop-close="true" class="badge-category clear-badge" title="Need help with your GitHub account or the core features of GitHub? This is the place to look for answers.">GitHub Help</span></a>
  </div>
  <blockquote>
    See this post, same problem: 
<a href="https://github.community/t/i-can-not-add-a-collaborator-to-the-private-repo/204664" class="onebox" target="_blank" rel="noopener">https://github.community/t/i-can-not-add-a-collaborator-to-the-private-repo/204664</a> 
if you join an existing thread your chances of finding a solution are higher, since you might discuss the issue with other uses who are experiencing the same problem, and because older posts are more likely to receive staff support first.
  </blockquote>
</aside>

**Comment by Yakimochkina** [comment, marked as answer]
<p>The problem is solved</p>

**Reply by sutexdevteam** [reply]
Hi Yakimochkina, how do you solve this problem?

**Reply by kripa-vkover** [reply]
Type their git email id. You will be able to add.

**Reply by techZone19** [reply]
I am still not able to add them b adding git ids

**Reply by ghostinhershell** [reply]
Hey @Yakimochkina! Thank you for confirming that this is resolved. The community would appreciate any guidance you can share on how you resolved the issue. This could help someone else who runs into this as well.

**Comment by joshgura** [comment]
tried to invite a collaborator (only one) to a private project, the guest clicks the invite and is not made a collaborator and is shown the door.
